src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/UpdateWorkitemProcedureStepState.sql

```diff
@@ -119,17 +119,14 @@ BEGIN
         WHERE
             cte.TagPath = @procedureStepStateTagPath
 
-        -- Update the Workitem status
-        EXEC dbo.UpdateWorkitemStatus @partitionKey, @workitemKey, @status
-
     END TRY
     BEGIN CATCH
 
         THROW
 
     END CATCH
 
-    SELECT @workitemKey
+    SELECT @@ROWCOUNT
 
```

Comment:
Are we using this value?

Reply:
Should we throw if the Rowcount is 0?

Reply:
We are not doing any of that right now. May be, we could throw if nothing is updated (ie., @@ROWCOUNT = 0)

Reply:
Done.